### PR TITLE
docs: add a link to packages.md

### LINKED ISSUE #1285: docs: improve links to documentation about required files
In feedback via email:

> CURRENT: Keyman Developer standard keyboard source file layout
> 
> REVISED: Keyman Developer structure for standard keyboard source file folders
> 
> I also wonder whether your link about distributing keyboards to Keyman
> applications (2nd link below) ought to have a link to the rather important
> include/exclude list in the 3rd link below.
> 
> ---
> 
> 1. https://help.keyman.com/developer/current-version/reference/file-layout
> 2. https://help.keyman.com/developer/17.0/guides/distribute/packages#toc-package-file-contents and
> 3. https://help.keyman.com/developer/keyboards/github/step-2
>

### PR DESCRIPTION
**Pull request by Nnyny**
fixes #1285

### CODE REVIEW

File: developer/17.0/guides/distribute/packages.md [hunk 1]
```diff
@@ -87,6 +87,10 @@ kmp.json
     be available only within the Keyman app and the on-screen-keyboard,
     not in other apps.
 
+> ### Tip
+Read [Step 2: Organizing the Keyboard Files](https://help.keyman.com/developer/keyboards/github/step-2)
```
Comment (darcywong00) [inline developer/17.0/guides/distribute/packages.md:91]: This path can be relative. Can you see if this works?

```suggestion
Read [Step 2: Organizing the Keyboard Files](../../../keyboards/github/step-2)
```

Comment (Nnyny) [reply developer/17.0/guides/distribute/packages.md:91]: Sorry @darcywong00, I couldn't test this because my docker isn't working yet.

Comment (darcywong00) [reply developer/17.0/guides/distribute/packages.md:91]: You should be able to test after merging in the master branch to get the fix from #1317

Comment (Nnyny) [reply developer/17.0/guides/distribute/packages.md:91]: The link works as expected.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**